cluster/manifests/skipper/deployment.yaml

```diff
@@ -457,6 +458,7 @@ spec:
           - "-kubernetes-backend-traffic-algorithm={{ .ConfigItems.skipper_ingress_backend_traffic_algorithm }}"
           - "-address=:9990"
           - "-wait-for-healthcheck-interval={{ .Cluster.ConfigItems.skipper_wait_for_healthcheck_interval }}"
+          - "kubernetes-default-lb-algorithm={{ .ConfigItems.skipper_ingress_default_lb_algorithm }}"
```

Comment:
Flag is is missing dash.
Could you please place this flag next to the previously introduced `skipper_ingress_backend_traffic_algorithm`